lib/admin/components/UserAdmin.js

```diff
@@ -66,30 +83,104 @@ export default class UserAdmin extends Component<Props> {
 
     // load orgs because they're needed both in org and user creation
     fetchOrganizations()
+
+    // load servers if deployments are enabled.
+    if (isModuleEnabled('deployment')) fetchServers()
   }
 
-  render () {
+  _getMainContent = (isApplicationAdmin: boolean): React.Node => {
     const {
       activeComponent,
       createUser,
       deleteUser,
       fetchProjectFeeds,
       fetchUsers,
       organizations,
+      otpServers,
       projects,
       setUserPage,
       setUserQueryString,
       updateUserData,
       user,
       users
     } = this.props
+    switch (activeComponent) {
+      case 'users':
+        if (!users.data || !projects || !organizations.data) {
+          return <Loading />
+        } else {
+          return (
+            <UserList
+              createUser={createUser}
+              creatingUser={user}
+              deleteUser={deleteUser}
+              fetchProjectFeeds={fetchProjectFeeds}
+              fetchUsers={fetchUsers}
+              isFetching={users.isFetching}
+              organizations={organizations.data}
+              page={users.page}
+              perPage={users.perPage}
+              projects={projects}
+              setUserPage={setUserPage}
+              setUserQueryString={setUserQueryString}
+              token={user.token || 'token-invalid'}
+              updateUserData={updateUserData}
+              userCount={users.userCount}
+              users={users.data}
+            />
+          )
+        }
+      case 'organizations':
+        if (!isApplicationAdmin || isModuleEnabled('enterprise')) return null
+        else return <OrganizationList {...this.props} />
```

Comment:
Probably sending even more props than are actually needed. The OrganizationList should be a connected component.